perfkitbenchmarker/openstack/os_disk.py

```diff
@@ -0,0 +1,85 @@
+import logging
+import time
+
+from perfkitbenchmarker.openstack import utils as os_utils
+from perfkitbenchmarker import flags
+from perfkitbenchmarker import disk
+from perfkitbenchmarker.openstack import utils
+from perfkitbenchmarker.openstack.utils import retry_authorization
+
+
+FLAGS = flags.FLAGS
+
+
+class OpenStackDisk(disk.BaseDisk):
+    characters = (c for c in "cdefghijklmnopqrstuvwxyz")
```

Comment:
rename to `virtual_disks`? In general, for variables that aren't just used in a small scope, the name should help explain the semantics of the values and how it gets used. In this case the interesting bit is that these are /dev/vdX names, not that they happen to be charactes.